### PR TITLE
:lipstick: datetime plugin to stick severine project

### PR DESCRIPTION
**Pull request by bpetetot**
image

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -5,73 +5,84 @@
...
+          period = 'matin'
+        } else if (hour >= 12 && hour < 14)  {
+          period = 'midi'
+        } else if (hour >= 14 && hour < 18)  {
+          period = 'après-midi'
+        } else if (hour >= 18 && hour < 22)  {
+          period = 'soir'
+        } else if (hour >= 22 && hour < 6)  {
+          period = 'nuit'
+        }
```
Comment (EmrysMyrddin) [inline frontend/index.html:61]:
```es6
        let period
        if (hour >= 22 || hour < 6)  {
          period = 'nuit'
        } else if (hour < 12)  {
          period = 'matin'
        } else if (hour < 14)  {
          period = 'midi'
        } else if (hour < 18)  {
          period = 'après-midi'
        } else if (hour < 22)  {
          period = 'soir'
        }
```

**Review by EmrysMyrddin — CHANGES_REQUESTED** [review]

File: frontend/index.html [hunk 2]
```diff
@@ -5,73 +5,84 @@
...
-    <span class="date">
-      <span id="day"></span>
-      <span id="month"></span>
-      <span id="year"></span>
-    </span>
+    </div>
   </div>
 
   <script>
     const leftPad = number => ('0' + number).slice(-2)
```
Comment (nlepage) [inline frontend/index.html]: I know this is not part of your PR, but we could use the [standardized padStart](https://mdn.io/String/padStart)

File: frontend/styles.css [hunk 3]
```diff
@@ -4,31 +4,30 @@ body {
...
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  flex-direction: column;
 }
 
 .date {
-  font-size: 1.2em;
-  font-weight: 300;
+  font-size: 5em;
```
Comment (nlepage) [inline frontend/styles.css:20]: Does this scale with the container ?

File: frontend/styles.css [hunk 4]
```diff
@@ -4,31 +4,30 @@ body {
...
+  font-size: 3em;
+  font-weight: 300;
 }
 
-@keyframes blinker {
-  50% {
-    opacity: 0;
-  }
+.time {
+  margin-top: 80px;
```
Comment (nlepage) [inline frontend/styles.css]: Why use px while other sizes are in em ?

File: frontend/styles.css (hunk 3 above)
Comment (bpetetot) [reply frontend/styles.css:20]: I changed it to make it scale following the viewport width (keeping a fallback to browser that does not support the feature)
```css
.date {
  font-size: 5em;
  font-size: 10vw;
  font-weight: 400;
}
```

**Review by EmrysMyrddin — APPROVED** [review]

**Pull request merged**